src/solver/conv_winoRxS_f2x3.cpp

```diff
@@ -855,15 +858,18 @@ ConvBinWinogradRxSf2x3::GetSolution(const ConvolutionContext& params,
     return result;
 }
 
-bool ConvBinWinogradRxSf2x3g1::IsApplicable(const ConvolutionContext& params) const
+bool ConvBinWinogradRxSf2x3g1::IsApplicable(const boost::any& ctx_) const
 {
+    auto params = boost::any_cast<const ConvolutionContext&>(ctx_);
+
     if(miopen::IsDisabled(MIOPEN_DEBUG_AMD_WINOGRAD_RXS_F2X3_G1{}))
         return false;
     return IsApplicableBase(params) && params.group_counts == 1;
 }
 
-float ConvBinWinogradRxSf2x3g1::GetWti(const ConvolutionContext& params) const
+float ConvBinWinogradRxSf2x3g1::GetWti(const boost::any& ctx_) const
 {
+    auto params = boost::any_cast<const ConvolutionContext&>(ctx_);
```

Comment:
A mixin should really be used to this casting instead of requiring every solver to do this.

Reply:
Code was changed as you proposed